helpers/types.bal

```diff
@@ -76,3 +76,19 @@ public type ai_spelling_responseOk record {|
     *http:Ok;
     ai_spelling_response body;
 |};
+
+public type curreny_converter_payload record {
+    string fromCurrency;
+    string toCurrency;
+};
+
+public type curreny_converter_response record {
```

Comment:
Can we make this a close record?

Postel's Law - "Be liberal in what you accept, and conservative in what you send."

Reply:
Fixed with 6c351e3